src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionTarget.js

```diff
@@ -36,6 +42,31 @@ const TextWrappper: ThemedComponent<{}> = styled.div`
   flex-grow: 1;
 `;
 
+const getTargetIcon = (currency?: TokenCurrency | CryptoCurrency) => {
+  if (!currency) return null;
+
+  return <CryptoCurrencyIcon circle currency={currency} size={16} />;
+};
+
+const AccountSection = ({
+  targetName,
+  targetCurrency,
+}: {
+  targetName: string,
+  targetCurrency: TokenCurrency | CryptoCurrency,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <SummarySection>
+      <SummaryLabel label={t("swap2.form.details.label.target")} />
+      <SummaryValue value={targetName} handleChange={() => {}}>
+        {getTargetIcon(targetCurrency)}
```

Comment:
Why not simply 
```
{targetCurrency? <CryptoCurrencyIcon circle currency={targetCurrency} size={16} /> : null}
```

Reply:
To improve clarity I use to limit computation/conditional from the JSX. From my mind, JSX must be simple to read and understand, avoiding complex map computations or complex conditionals is a good practice.

In this case, it's completely unrelated and I agree with you that I prefer a ternary than calling a function here. I'm going to update the code.

Reply:
If there's heavy lifting for sure, but in this particular case I think it's simple enough

Reply:
Apply change there [0293d1eb832852b8c0e977a1cb975bcba69688ab](https://github.com/juan-cortes/ledger-live-desktop/pull/14/commits/0293d1eb832852b8c0e977a1cb975bcba69688ab)